src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -396,23 +396,12 @@ def load_command_table(self, _):
             "aks_mesh_enable_ingress_gateway",
             supports_no_wait=True,
         )
-        g.custom_command(
-            "enable-egress-gateway",
-            "aks_mesh_enable_egress_gateway",
```

Comment:
Oh, I think you should also delete the command implementations, like function `aks_mesh_disable_egress_gateway` in `custom.py`

Reply:
Would like to keep this if possible as we will likely reintroduce egress gateway in the near future.

Reply:
I agree with Niranjan here, we would like to keep it as we see that egress gateway functionality will be reintroduced. This PR is to just remove user/test facing code components. @FumingZhang let us know your thoughts?

Reply:
ok, get it